security/container_security/security-platform.adoc

```diff
@@ -0,0 +1,46 @@
+[id="security-platform"]
+= Securing the container platform
+include::modules/common-attributes.adoc[]
+:context: security-platform
+
+toc::[]
+
+{product-title} and Kubernetes APIs are key to automating container management at scale. APIs are used to:
+
+- Validate and configure the data for pods, services, and replication controllers.
+- Perform project validation on incoming requests and invoke triggers on other
+major system components.
+
+Security-related features in {product-title} that are based on Kubernetes include:
+
+* Multitenancy, which combines Role-Based Access Controls and network policies
+to isolate containers at multiple levels.
+* Admission Controllers, which form boundaries between an API and those
+making requests to the API.
+
+{product-title} uses Operators to automate and simplify the management of
+Kubernetes-level security features.
+
+// Multitenancy
+include::modules/security-platform-multi-tenancy.adoc[leveloffset=+1]
+
+// Admission plug-ins
+include::modules/security-platform-admission.adoc[leveloffset=+1]
+
+// Authentication and authorization
+include::modules/security-platform-authentication.adoc[leveloffset=+1]
+
+// Managing certificates for the platform
+include::modules/security-platform-certificates.adoc[leveloffset=+1]
+
+.Additional resources
+- xref:../../welcome/oke_about.adoc#oke-about[About {product-title}]
+- xref:../../authentication/using-rbac.adoc#using-rbac[Using RBAC to define and apply permissions]
+- xref:../../architecture/admission-plug-ins.adoc#admission-plug-ins[About Admission plug-ins]
+- xref:../../authentication/managing-security-context-constraints.adoc#managing-pod-security-policies[Managing Security Context Constraints]
+- xref:../../authentication/managing-security-context-constraints.adoc#security-context-constraints-command-reference_configuring-internal-oauth[SCC Reference Commands]
+- xref:../../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-managing_understanding-service-accounts[Examples of granting roles to service accounts]
```

Comment:
This goes to the section above the one you want. Anchor needs updated

Reply:
Good catch. Fixed.